CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ The big thing to look for will be that the persons name is spelled correctly in
 
 #### Using Labels
 
-* **Work in Progress** - This should be applied if the pull request should NOT be accepted in its current state. This is for WIP PR's that require more work from the contributor
+* **Work in Progress** - This should be applied if the pull request should **NOT** be accepted in its current state. This is for WIP PR's that require more work from the contributor
```

Comment:
* **Work in Progress** - This will be applied if the pull request (PR) should **NOT** be accepted in its current state. This is for  work-in progress PR's that require more work from the contributor.

Reply:
Added